vagrant/2/1/README.md

```diff
@@ -0,0 +1,6 @@
+Use:
```

Comment:
I would suggest you use the README file in vagrant/1/1 as a template. Just copy it and add the status and ssh commands.

Reply:
Thanks for the feedback.